includes/phing/build/main/subsite.xml

```diff
@@ -32,6 +32,16 @@
             subsite-resources-copy,
             package-version-set,
             project-subsite-composer-no-dev">
+        <if>
```

Comment:
This should be include in a new target `build-dist` to make sure we don't get conflicts with projects overriding `build-subsite-dist`